src/services/intervieweeService.js

```diff
@@ -28,9 +28,37 @@ exports.getInterviewee = async (intervieweeId) => {
 exports.getInterviewees = async (projectId) => {
   try {
     const { candidates } = await Project.findOne({ _id: projectId }).populate("candidates").lean();
-
+    
     return { candidates };
   } catch (error) {
     return { error };
   }
 };
+
+exports.updateInterviewee = async ({ intervieweeId, interviewee }) => {
+  try {
+    const intervieweeData = await Interviewee.findOne({ _id: intervieweeId });
+    
+    Object.keys(interviewee.filterScores).forEach((key) => {
+      if (!intervieweeData.filterScores[key]) {
+        intervieweeData.filterScores[key] = [];
+      }
+      
+      intervieweeData.filterScores[key].push(interviewee.filterScores[key]);
+    });
```

Comment:
지금 작업도 맞는 방법이지만 가져온 intervieweeData안에 있는 filterScores를 직접적으로 수정하기보다는 
빼내서 작업한 후 이후의 update작업을 진행하는게 더 좋을 것 같아요~!

```suggestion
    const newFilterScores = [...interviewee.filterScroes];
    
    Object.keys(interviewee.filterScores).forEach((key) => {
      if (!newFilterScores[key]) {
        newFilterScores[key] = [];
      }
      
      newFilterScores[key].push(interviewee.filterScores[key]);
    });
```